graphql/mutation_resolver.go

```diff
@@ -371,6 +371,16 @@ func (r *mutationResolver) AttachProjectToRepo(ctx context.Context, projectID st
 func (r *mutationResolver) CreateProject(ctx context.Context, project restModel.APIProjectRef) (*restModel.APIProjectRef, error) {
 	dbProjectRef := project.ToService()
 	u := gimlet.GetUser(ctx).(*user.DBUser)
+
+	config, err := evergreen.GetConfig()
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting evergreen config: %s", err.Error()))
+	}
+
+	if err := dbProjectRef.ValidateOwnerAndRepo(config.GithubOrgs); err != nil {
```

Comment:
Can we just add both of these statements to data.CreateProject since both places need to check it? (I think it's fine to re-get the config in the UI case rather than using the passed in one)

Reply:
No need to block on another review from me when this is done

Reply:
Done. I'm assuming you meant that it's okay to get the config in the rest case?